src/pzpr/variety.js

```diff
@@ -286,6 +286,7 @@
 			sudoku: [0, 1, "数独", "Sudoku"],
 			sukoro: [1, 0, "数コロ", "Sukoro", "sukoro"],
 			sukororoom: [0, 0, "数コロ部屋", "Sukoro-room", "sukoro"],
+			symmarea: [0, 1, "シンメトリーエリア", "Symmetry Area", "fillomino"],
```

Comment:
the 1 seems inaccurate, or does kanpen support this type?

Reply:
Ah, this is what happens when I blindly copy-paste what's written for Fillomino, and forget to check what the values mean. I doubt this is supported by kanpen.